spec/controllers/contact_controller_spec.rb

```diff
@@ -48,8 +50,7 @@
   end
   context 'with contact e-mail set' do
     before do
-      AppConfig.first
-        .update_attributes(contact_link_location: 'contact@example.com')
+      allow(@ac).to receive(:contact_link_location) { 'contact@example.com' }
```

Comment:
This is great! I think I'd prefer the syntax `allow(@ac).to receive(:contact_link_location).and_return('contact@example.com')` - it reads a lot better to me (and was how stubbing was first presented to me). What do you think?